api/MediaSource.json

```diff
@@ -154,7 +154,7 @@
               {
                 "version_added": "23",
                 "version_removed": "31",
-                "prefix": "-webkit-"
+                "prefix": "webkit"
```

Comment:
2. I'm wondering if the prefixes on methods is spurious. In other words, is this really, `webkitMediaSource.webkitSourceBuffers` or is it `webkitMediaSource.sourceBuffers`? That bug link implies the latter, but I'm not quite sure

(Both of these questions apply throughout the PR)